src/vos/evtree.c

```diff
@@ -2540,14 +2540,19 @@ evt_ent_array_fill(struct evt_context *tcx, enum evt_find_opc find_opc,
 				 * place so I did it this way to minimize
 				 * change while we decide how to handle this
 				 * properly.
+				 * RT_OVERLAP_INCLUDED/RT_OVERLAP_INCLUDES is possible
+				 * when vos aggregation merged exts between retried
+				 * rebuild, returned DER_RECX_OVERLAP for that case.
 				 */
 				if (range_overlap != RT_OVERLAP_SAME) {
-					D_ERROR("Same epoch partial "
-						"overwrite not supported:"
-						DF_RECT" overlaps with "DF_RECT
-						"\n", DP_RECT(rect),
-						DP_RECT(&rtmp));
-					rc = -DER_NO_PERM;
+					if (range_overlap == RT_OVERLAP_INCLUDED ||
+					    range_overlap == RT_OVERLAP_INCLUDES)
+						rc = -DER_RECX_OVERLAP;
+					else
+						rc = -DER_NO_PERM;
+					D_ERROR("Same epoch partial overwrite not supported:"
```

Comment:
Perhaps this should be
D_CDEBUG(rc == -DER_NO_PERM, DLOG_ERR, DB_IO, ....)

Since -DER_RECX_OVERLAP is expected to happen.